server.js

```diff
@@ -1,6 +1,7 @@
 /**
  * Module dependencies.
  */
+require('dotenv').config();
```

Comment:
I didn't add that require statement, it was part of the original codebase. for the commented code, I'd fix that.